include/mpm_explicit.tcc

```diff
@@ -537,3 +537,249 @@ void mpm::MPMExplicit<Tdim>::write_vtk(mpm::Index step, mpm::Index max_steps) {
   }
 }
 #endif
+
+//! MPM Explicit solver
+template <unsigned Tdim>
+bool mpm::MPMExplicit<Tdim>::solve() {
+  bool status = true;
+  
+  // Get anlysis type USL/USF
+  bool USL = false;
```

Comment:
Remove this as it is set in the construction stage. See previous edit in the header file